lightning/src/ln/peer_handler.rs

```diff
@@ -1231,25 +1275,49 @@ impl<Descriptor: SocketDescriptor, CM: Deref, RM: Deref, L: Deref, CMH: Deref> P
 	/// [`ChannelManager::process_pending_htlc_forwards`]: crate::ln::channelmanager::ChannelManager::process_pending_htlc_forwards
 	/// [`send_data`]: SocketDescriptor::send_data
 	pub fn process_events(&self) {
+		let mut _single_processor_lock = self.event_processing_lock.try_lock();
```

Comment:
nit: no leading underscore on the var

Reply:
Sadly this leads to a `warning: value assigned to .... is never read`